### PR TITLE
[CLI]Fix CLI

### PR DESCRIPTION
**Pull request by ashishb**
### Description

Before this fix,
```
packages/cli $ yarn run celocli account:new
yarn run v1.17.3
$ /Users/ashishb/celo/celo-monorepo5/node_modules/.bin/celocli account:new
TypeError: Cannot read property 'modules' of undefined
    at Function.generateAccountAddressFromPrivateKey (~/celo/celo-monorepo5/packages/cli/src/commands/account/new.ts:34:21)
    at NewAccount.run (~/celo/celo-monorepo5/packages/cli/src/commands/account/new.ts:42:39)
    at NewAccount._run (~/celo/celo-monorepo5/node_modules/@oclif/command/lib/command.js:44:31)
    at <anonymous>
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

After this fix,
```
packages/cli $ yarn run celocli account:new
yarn run v1.17.3
$ /Users/ashishb/celo/celo-monorepo5/node_modules/.bin/celocli account:new
This is not being stored anywhere, so, save the mnemonic somewhere to use this account at a later point

mnemonic: novel thank tattoo gaze unusual life prefer warfare suspect need must novel predict thought praise assist laundry napkin please alarm token video galaxy angry
privateKey: b0260f30c168bec4af8652fcab7065431faf37c8a2bb3cc6e7f75ea370659b66
publicKey: 04693e72d10685fdee703cfa73e35f0ae73f0ecbd46522202c65cc35896311cdc3d3d9b697692c27b55e09f95e6f9a08f8fad196288d44e5db16f0a130bf2f6b51
accountAddress: 0xBc1B53db25cb0197510F6F946384C632aF9e64ea
✨  Done in 1.26s.
```

### CODE REVIEW

**Review by tkporter — APPROVED** [review]

File: packages/cli/tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "extends": "../typescript/tsconfig.library.json",
   "compilerOptions": {
     "rootDir": "src",
-    "outDir": "lib"
+    "outDir": "lib",
+    "esModuleInterop": true
```
Comment (mcortesi) [inline packages/cli/tsconfig.json:6]: we should place this in `../typescript/tsconfig.library.json`.

So all of our packages share the same config

**Review by mcortesi — CHANGES_REQUESTED** [review]

File: packages/cli/tsconfig.json (hunk 1 above)
Comment (ashishb) [reply packages/cli/tsconfig.json:6]: It is already in that file but it is not helping: https://github.com/celo-org/celo-monorepo/blob/master/packages/typescript/tsconfig.library.json#L11

Comment (mcortesi) [reply packages/cli/tsconfig.json:6]: that's awkward...

try removing it and run `tsc --showConfig` and see if it is there

Comment (ashishb) [reply packages/cli/tsconfig.json:6]: It is there :/

Comment (ashishb) [reply packages/cli/tsconfig.json:6]: I am getting 

```
 $ yarn run celocli account:new
yarn run v1.17.3
$ /Users/ashishb/celo/celo-monorepo5/node_modules/.bin/celocli account:new
/Users/ashishb/celo/celo-monorepo5/packages/contractkit/lib/base.js:19
exports.AllContracts = Object.keys(CeloContract).map(function (k) { return CeloContract[k]; }, exports.as, any), exports.as = CeloContract[];
                                                                                                                                           ^
SyntaxError: Unexpected token ]
    at Object.<anonymous> (~/celo/celo-monorepo5/packages/contractkit/src/index.ts:3:1)
    at Object.<anonymous> (~/celo/celo-monorepo5/packages/cli/src/base.ts:1:1)
    at Module.m._compile (~/celo/celo-monorepo5/node_modules/ts-node/src/index.ts:473:23)
    at Object.require.extensions.(anonymous function) [as .ts] (~/celo/celo-monorepo5/node_modules/ts-node/src/index.ts:476:12)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

So, I will have to debug that for testing further.

**Review by mcortesi — APPROVED** [review]
I've checked, and it seems that we need to add the config option to tsconfig.... weird...

maybe a ts-node thing

**Pull request merged**